src/codechicken/nei/recipe/GuiRecipeTab.java

```diff
@@ -177,7 +181,13 @@ public static void loadHandlerInfo() {
         NEIClientConfig.logger.info("Loading handler info from " + (fromJar ? "JAR" : "Config"));
         handlerMap.clear();
         URL handlerUrl = Thread.currentThread().getContextClassLoader().getResource("assets/nei/csv/handlers.csv");
-        
+        handlerMap.putAll(handlerAdderFromIMC);
+        handlerMap.forEach((s, h) -> {
```

Comment:
Why not something like `handlerMap.keySet().removeAll(handlerRemoverFromIMC);`

Reply:
haha, this also won't actually do anything given it's being run right after the clear, and before the CSV is loaded... These should be at the end of the function/called after it